### PR TITLE
Add Experimental Design for Multi-Channel Imaging via Task-Driven Feature Selection

### PR DESCRIPTION
**Pull request by sbb-gh**

### CODE REVIEW

**Comment by robmarkcole** [comment]
Rather than linking the pdf (which will slow cicd) could you use https://openreview.net/forum?id=MloaGA6WwX ?

**Comment by sbb-gh** [comment]
Completed

File: README.md [hunk 1]
```diff
@@ -1594,7 +1594,7 @@ A variety of techniques can be used to count animals, including object detection
 - [Pytorch Benchmarks for Object Detection in Aerial Images](https://github.com/dingjiansw101/AerialDetection)
 
 - [ASPDNet](https://github.com/liuqingjie/ASPDNet) -> Counting dense objects in remote sensing images
-
+/
```
Comment (robmarkcole) [inline README.md:1597]: typo

Comment (sbb-gh) [reply README.md]: Done

**Pull request merged**